deploy/medulla/templates/vs.yaml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Dont believe we need this if were using the istio ingress class no? You can either use [Virtual Services + Gateways](https://istio.io/latest/docs/reference/config/networking/virtual-service/) or the [Istio Ingress Class](https://istio.io/latest/docs/tasks/traffic-management/ingress/kubernetes-ingress/) but not both.

Reply:
If the VS and gateway are not needed then that will allow us to use only one helm chart. Do you want me to move forward with that?

Reply:
VS and Gateways have more features than an Ingress, so I would prefer we just use the Istio Gateway and VS's but there does need to be a VS for all services that are to be exposed.

You will still need two charts to support other dependencies being deploy (ex. TiDB cluster)